core/core-api/src/main/java/com/hbc/pms/core/api/task/PlcPublisher.java

```diff
@@ -0,0 +1,41 @@
+package com.hbc.pms.core.api.task;
+
+import com.hbc.pms.core.api.constants.StationEnum;
+import com.hbc.pms.core.api.service.StationService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PlcPublisher {
```

Comment:
why is this one called publisher? I don't see any publishing logic here

Reply:
It is expected to publish events to Websocket connections. A POC has already been implemented by NHuy in this branch https://github.com/f-pms/API/commit/89b1de9947fbb9d2b1fda7adac42c3933ffea2db#diff-0fe483bf029447b5fe95cd6291d9c195f2a642619afe4d8cac1d8f0c98ef095a

Reply:
It will be implemented by @thinhle19